docs/en-us/dev/user_doc/guide/alert/enterprise-wechat.md

```diff
@@ -1,29 +1,9 @@
 # Enterprise WeChat
 
-If you need to use Enterprise WeChat to alert, please modify the `alert.properties` file after the installation is complete, and then restart the alert service. The configuration example of enterprise WeChat is as follows
+If you need to use Enterprise WeChat to alert, please create alarm Instance in warning instance manage, and then chose the wechat plugin. The configuration example of enterprise WeChat is as follows
 
-```shell
-# Enable enterprise WeChat alarm or not: ``true`` or ``false``
-enterprise.wechat.enable="true"
+<img src="enterprise-wechat-plugin.png">
```

Comment:
I thinks we should use markdown syntax here, `[enterprise-wechat-plugin](enterprise-wechat-plugin.png)`. WDYT
```suggestion
<img src="enterprise-wechat-plugin.png">
```